osu.Game.Tests/Visual/Multiplayer/TestSceneRankRangePill.cs

```diff
@@ -1,67 +1,68 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System.Collections.Generic;
+using System.Linq;
+using Moq;
 using NUnit.Framework;
+using osu.Framework.Allocation;
 using osu.Framework.Graphics;
 using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Online.Rooms;
 using osu.Game.Screens.OnlinePlay.Lounge.Components;
 
 namespace osu.Game.Tests.Visual.Multiplayer
 {
-    public class TestSceneRankRangePill : MultiplayerTestScene
+    public class TestSceneRankRangePill : OsuTestScene
     {
-        [SetUp]
-        public new void Setup() => Schedule(() =>
+        private readonly Mock<MultiplayerClient> multiplayerClient = new Mock<MultiplayerClient>();
+
+        protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnlyDependencyContainer parent) =>
+            // not used but required by the component.
```

Comment:
This comment threw me for a loop because I didn't know what it meant, but I think I got it eventually - would the following be an accurate explanation?

```suggestion
            // not used directly in component, but required due to it inheriting from OnlinePlayComposite.
```